environment.yml

```diff
@@ -8,21 +8,21 @@ dependencies:
   - pip
   # For packaging, and testing
   # - arcpy  # windows only
+  - build
   - flake8
   - flake8-import-order
   - flake8-blind-except
   - flake8-builtins
-  - setuptools
-  - setuptools_scm
-  - wheel
+  - setuptools>=61
+  - setuptools_scm>=6.2
   - pytest
   - pytest-console-scripts
   - pytest-cov
   # For running
   - astropy
   - fiona
   - gdal>=3.3
-  - numpy
+  - numpy<1.24.0
```

Comment:
There are a lot of depreciations in numpy 1.24 and it seems to be messing with a lot of packages (also will unblock #154); we'll want to open a PR that unrestricts NumPy and periodically run the actions until it passes.